include/xeus-cpp/xeus_cpp_config.hpp

```diff
@@ -14,6 +14,9 @@
 #define XEUS_CPP_VERSION_MINOR 4
 #define XEUS_CPP_VERSION_PATCH 0
 
+#define XCPP_TAGCONFS_DIR "../etc/xeus-cpp/tags.d"
```

Comment:
As I mentioned earlier, this file should become `xeus_cpp_config.hpp.in` and then we should configure it with cmake's `configure_file`, passing the correct locations.